<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Start</class>
 <widget class="QWidget" name="Start">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>350</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>350</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Quiz</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: rgb(51, 51, 51);</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>340</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>20</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>100</width>
     <height>20</height>
    </size>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
font-size: 13px;
        color: white;
border-radius: 1px;
background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:0, y2:0, stop:0 rgba(52, 52, 52, 255), stop:1 rgba(85, 85, 85, 255));
     }

QPushButton:hover{ background-color: qlineargradient(spread:pad, x1:0, y1:0.136364, x2:1, y2:0.602273, stop:0.0207254 rgba(255, 180, 0, 255), stop:1 rgba(187, 22, 77, 255));
font-size: 16px;
}


</string>
   </property>
   <property name="text">
    <string>Hot</string>
   </property>
  </widget>
  <widget class="QLabel" name="startlogo">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>50</y>
     <width>161</width>
     <height>121</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>340</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>20</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>100</width>
     <height>20</height>
    </size>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
font-size: 13px;
        color: white;
border-radius: 1px;
background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:0, y2:0, stop:0 rgba(52, 52, 52, 255), stop:1 rgba(85, 85, 85, 255));
     }

QPushButton:hover{ background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0.767, stop:0 rgba(153, 255, 255, 255), stop:1 rgba(153, 255, 131, 255));
color:rgb(51, 51, 51);
font-size: 16px;
}
</string>
   </property>
   <property name="text">
    <string>Cold</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>480</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; font-style:italic;&quot;&gt;Made by Sebastian OÅ›kiewicz&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>290</y>
     <width>350</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <weight>12</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
font-size: 13px;
        color: white;
font-weight: 100;
border-radius: 1px;
background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:0, y2:0, stop:0 rgba(52, 52, 52, 255), stop:1 rgba(85, 85, 85, 255));
     }

QPushButton:hover{ 
font-size: 16px;
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(204, 204, 204, 255), stop:1 rgba(255, 255, 255, 255));
color: rgb(51, 51, 51);
}


</string>
   </property>
   <property name="text">
    <string>START</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_tool">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>10</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border:none;
qproperty-icon: url(:/MojePliki/tool.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label_sound">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>280</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;
</string>
   </property>
   <property name="text">
    <string>SOUND</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_music">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>310</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;</string>
   </property>
   <property name="text">
    <string>MUSIC</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_color">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>340</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;</string>
   </property>
   <property name="text">
    <string>COLOR</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border:none;
qproperty-icon: url(:/MojePliki/back.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>pushButton_3</zorder>
  <zorder>pushButton</zorder>
  <zorder>startlogo</zorder>
  <zorder>pushButton_2</zorder>
  <zorder>label</zorder>
  <zorder>pushButton_tool</zorder>
  <zorder>label_sound</zorder>
  <zorder>label_music</zorder>
  <zorder>label_color</zorder>
  <zorder>pushButton_4</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
